starwars/src/components/CharCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { imgs } from '../img'
```

Comment:
ES6 import syntax is what all the cool kids are using. Try `import imgs from '../img'